main.go

```diff
@@ -31,4 +24,11 @@ func main() {
 	if err := server.ListenAndServe(); err != nil {
 		panic(err)
 	}
+
+	go func() {
+		for {
+			releaseCollector.UpdateFromCatalog()
+			time.Sleep(time.Minute * 30)
+		}
```

Comment:
@JosephSalisbury do you know a better way? problem with this is the build fails because -> https://app.circleci.com/pipelines/github/giantswarm/release-exporter/6/workflows/fb434ccc-ed83-42e0-8501-5661a0befb53/jobs/6

Reply:
it worked now :)